domino/helpers.py

```diff
@@ -0,0 +1,20 @@
+from distutils.version import LooseVersion as parse_version
+from .constants import MINIMUM_SUPPORTED_DOMINO_VERSION
+from version import PYTHON_DOMINO_VERSION
+
+
+def is_version_compatible(version: str) -> None:
```

Comment:
I would call this `enforce_version_compatibility` or `ensure_version_compatible` or something similar. In Java, if a function is called `isSomething`, it is assumed to be side-effect free and returning a Boolean. I know Python is not Java, but still...